examples/evaluators/golang/simple/main.go

```diff
@@ -47,30 +49,15 @@ func main() {
 
 	// Read config
 	lgr.Println("Initializing config...")
-	cfg, err := readConfig("matchmaker_config", map[string]interface{}{
-		"REDIS_SERVICE_HOST": "redis",
-		"REDIS_SERVICE_PORT": "6379",
-		"auth": map[string]string{
-			// Read from k8s secret eventually
-			// Probably doesn't need a map, just here for reference
-			"password": "12fa",
-		},
-	})
+	cfg, err := config.Read()
 	if err != nil {
 		panic(nil)
 	}
 
 	// Connect to redis
-	// As per https://www.iana.org/assignments/uri-schemes/prov/redis
-	// redis://user:secret@localhost:6379/0?foo=bar&qux=baz // redis pool docs: https://godoc.org/github.com/gomodule/redigo/redis#Pool
-	redisURL := "redis://" + cfg.GetString("REDIS_SERVICE_HOST") + ":" + cfg.GetString("REDIS_SERVICE_PORT")
-	lgr.Println("Connecting to redis at", redisURL)
-	pool := redis.Pool{
-		MaxIdle:     3,
-		MaxActive:   0,
-		IdleTimeout: 60 * time.Second,
-		Dial:        func() (redis.Conn, error) { return redis.DialURL(redisURL) },
-	}
+	pool := redisHelpers.ConnectionPool(cfg)
```

Comment:
This is using the old red ConnectionPool call site. There's an error also sent back.

https://github.com/GoogleCloudPlatform/open-match/pull/87
I think this PR needs to be synced.

Reply:
yeah I forgot about this, fixed now